src/contract/aci/transformation.js

```diff
@@ -319,14 +318,14 @@ export function transformDecodedData (aci, result, { skipTransformDecoded = fals
  * @param bindings
  * @return {Object} JoiSchema
  */
-export function prepareSchema (type, { bindings } = {}) {
+function prepareSchema (type, bindings) {
   const { t, generic } = readType(type, { bindings })
 
   switch (t) {
     case SOPHIA_TYPES.int:
-      return Joi.number().error(getJoiErrorMsg)
+      return Joi.number().unsafe()
```

Comment:
`unsafe()` is used because of the BigInt, right?

Reply:
We have places like https://github.com/aeternity/aepp-sdk-js/blob/a018bf7154e5da1683180bd663ed6f0643efca2a/test/integration/contract.js#L213
As I got, new Joi started to check if number potentially lost precision, this is a good thing but this is a breaking change for us, so I'm disabling it.